src/lib.rs

```diff
@@ -526,6 +530,17 @@ impl<'a> EventCtx<'a> {
     pub fn window(&self) -> &WindowHandle {
         &self.window
     }
+
+    /// Set the event as "handled", which stops its propagation to other
+    /// widgets.
+    pub fn set_handled(&mut self) {
```

Comment:
There's an asymmetry here between `set_handled` and `set_active`. I prefer this to not take an argument, but maybe `set_active` doesn't need to either?

Reply:
No, `set_active` can definitely take `false`, for example on mouse-up in a button. I don't think it makes any sense to unset handled. Open to ideas of improving naming/docs tho.